tests/AutoReview/FixerTest.php

```diff
@@ -168,6 +168,28 @@ public function testFixersAreDefined(FixerInterface $fixer)
         $this->assertInstanceOf(\PhpCsFixer\Fixer\DefinedFixerInterface::class, $fixer);
     }
 
+    /**
+     * @dataProvider provideFixerDefinitionsCases
+     */
+    public function testDeprecatedFixersHaveCorrectSummary(FixerInterface $fixer)
+    {
+        $reflection = new \ReflectionClass($fixer);
+
+        if (false !== strpos($reflection->getDocComment(), '@deprecated')) {
+            $this->assertRegExp(
+                '/DEPRECATED: [a-z]/',
```

Comment:
```
/Deprecated: use `[a-z_]+` instead.$/
```

Reply:
done, what about 2nd RegEx? I was thinking about making it case-insensitive, but summary of `ereg_to_preg` contains "deprecated" word and it should be there.

Reply:
case-insensitive variant would be less strict, thus less unified. and you wanted to unified it ;)

Reply:
2nd re is fine imo